fr.etnadji.dispremi.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "fr.etnadji.dispremi",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dispremi",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
What does it need host access for?

Reply:
The application convert files that could be on other disks.

Reply:
It should ideally use GtkNativeFileChooser which delegates filesystem access to a portal.